weblate/trans/views/reports.py

```diff
@@ -136,7 +141,10 @@ def get_credits(request, path=None):
                 translators=format_html_or_plain_join(
                     "\n",
                     translator_format,
-                    ((t[0], t[1], t[2]) for t in translators),
+                    (
+                        (t["email"], t["full_name"], t["change_count"])
+                        for t in translators
+                    ),
```

Comment:
When we already have dict with values, it makes sense to change the above format strigs to use named placeholders and format using the dict directly.

Reply:
I'm not sure I can change the `format_plaintext_join` and `format_html_join` methods to accept dicts due to their use of generators

Reply:
Ah, I didn't realize that `format_html_join` doesn't support kwargs. Without that my comment isn't really meaningful.